browser.json

```diff
@@ -234,6 +234,6 @@
       "from": "./dist/wizard-stepper/ds6/wizard-stepper.css",
       "to": "./dist/wizard-stepper/ds4/wizard-stepper.css",
       "if-flag": "ds-4"
-    },
```

Comment:
Leave this change out. Since this was merged into master it will cause conflicts. 
I will update `10.4.0` with changes from master

Reply:
Code-gen will break if this file is not json compliant.

Reply:
Yes but this has already been fixed. This will cause a merge conflict and it's been already fixed. https://github.com/eBay/skin/blob/10.4.0/browser.json#L237
Not sure what is going on here since this is been fixed both on master and `10.4.0`

Reply:
Possibly this branch needs rebasing with 10.4.0 then this file will disappear from this PR.

Reply:
@raveenakothapally Can you rebase your branch with the latest from 10.4.0, then force push your branch? Or I can do it. Should be quick.

Reply:
I'm going to quickly rebase this then push and merge.